CHANGELOG.md

```diff
@@ -1,8 +1,9 @@
-## 0.7.11-dev2
+## 0.7.11-dev3
```

Comment:
```suggestion
## 0.7.11
```

Reply:
I assume you meant to cut a release with this merge?